apps/key-manager/src/components/auth/SignOut.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import { DropdownMenuItem } from "@/components/ui/dropdown-menu";
+import { LogOutIcon } from "lucide-react";
+import { signOut } from "next-auth/react";
+
+export default function SingOut() {
+  function handleSignOut() {
+    signOut({
+      redirect: true,
+      redirectTo: "/",
```

Comment:
It may be best to redirect straight to `/login` (which is also easier for me to test locally).